guides/10-dca.md

```diff
@@ -0,0 +1,154 @@
+- On Day 3, you will receive $300 USDC worth of SOL in your wallet. If the price of SOL stay stagnant and remain as $25 USDC / SOL, you will receive 11.988 SOL *(net of fees)* in your wallet.
+
+:::tip Caveat to auto-withdrawal
+In the event your desired purchased token is not SOL, auto withdrawal for each order will only work if user have the correct associated token account (ATA) open and ready.
+:::
+
+- By default, Jupiter DCA open the necessary ATA for your wallet when you create a Jupiter DCA account. However, if you close your purchased token's ATA manually via wallet interface or any other 3rd-party tool, tokens will not transferred to you automatically on every order but only at the end of your DCA cycles as a single lumpsum amount. **This only applicable to SPL tokens and not SOL.**
+
+- An example, if you have a DCA order into BONK with $300 USDC over 3 days, but if you choose to close your wallet's BONK token account before the 2nd trade/ transaction, the last trade/ transaction of $100 USDC worth of BONK will remain in the program account. You can choose to withdraw BONK from the program account at anytime through our UI.
+
+- During your last trade/ transaction, auto-withdrawal will happen regardless of whether you have a BONK ATA or not. Based on the example above, even though your BONK tokens does not get transferred to your wallet automatically for your 2nd trade/ transaction, your BONK tokens will be transferred to your wallet on the very last trade - see auto close below for more information.
+
+---
+
+### Auto Close
+
+:::info Auto Close
+At the end of your DCA orders, you would not need to perform any actions, any remaining tokens and rent is transferred to your wallet account directly.
+:::
+
+By default, if your wallet’s ATA remains open, your purchase tokens are automatically transferred to you on every order (the program only allows your tokens to be sent to your wallet and no one else’s). 
+
+However, in the event, there are remaining purchase tokens yet to be transferred to you (ie. if you close your wallet’s token account for your purchase token halfway through the DCA cycle as described in auto-withdrawal section), the DCA program will close the PDA in-token account and utilize the rent recovered to open the correct ATA for your wallet to store your purchased token. 
+
+This allows the program to transfer the remaining purchased tokens to you. As such, you would only receive 2/3 of the rent incurred to initialise your DCA account. Having said that, this 1/3 rent is not absorbed by any third-party. This rent will remain recoverable by you if you decide to close your ATA that “holds” your purchased token. 
+
+:::warning 
+Do not close your ATA without first withdrawing the tokens or swapping them for other tokens. Doing so can result in the lost of tokens. This is not unique to Jupiter or Jupiter’s DCA. This has to do with how Solana wallets and accounts work.
```

Comment:
Taking account of the above, in what cases is a loss of token possible?

Reply:
@Timoon21 Won't happen. It's just a matter of when they receive the tokens - after every single order or at the very end.